seed/static/seed/partials/inventory_list.html

```diff
@@ -45,7 +45,7 @@ <h1>{$:: (inventory_type === 'taxlots' ? 'Tax Lots' : 'Properties') | translate
                     <option value="update_salesforce" ng-disabled="!organization.salesforce_enabled || selectedCount === 0" translate>Update Salesforce</option>
                 </optgroup>
                 <optgroup label="{$:: 'Options' | translate $}">
-                    <option value="open_show_populated_columns_modal" ng-disabled="data.length === 0" translate>Only Show Populated Columns</option>
+                    <option value="open_show_populated_columns_modal" ng-if="menu.user.is_ali_root" ng-disabled="data.length === 0" translate>Only Show Populated Columns</option>
```

Comment:
is this getting hidden b/c it essentially updates a column list profile?

Reply:
yep!